src/pyasm/unittest/environment.py

```diff
@@ -50,11 +50,18 @@ def create(self):
     def delete(self):
         print("Deleting existing Unittest project")
         related_types = ["sthpw/schema", "sthpw/task","sthpw/snapshot", "sthpw/file"]
+        self.commit()
```

Comment:
Can you test if DeleteProjectCommand works on MySQL? If it does not, then we might need to move the commit into DeleteProjectCommand.